terraform/cloudfoundry/outputs.tf

```diff
@@ -11,7 +11,11 @@ output "ssh_elb_name" {
 }
 
 output "cf_root_domain" {
```

Comment:
They're not the same value.  `cf_root_domain` includes the environment component.